README.md

```diff
@@ -1 +1,41 @@
-# lboard
\ No newline at end of file
+# lboard
+
+In this project, I have built a web app powered by the Leaderboard APIs. I've achieved by using the Webpack Framework.
+
+## Live Link
+
+[Live Link](https://softwareengineermwi.github.io/lboard/)
```

Comment:
- [ ] **OPTIONAL** Please provide a valid live demo link to the README file or you could remove it since it's not required for this project.